src/components/molecules/card/card.scss

```
@@ -0,0 +1,53 @@
+@use "../../variables" as *;
+
+.card{
+    display: inline-block;
+    border-radius: 0;
+    overflow: hidden;
+    border: none;
+
+    .card-top{ 
+        position: relative;
+        overflow: hidden;
+
+        .card-title-wrapper{
+            position: absolute;
+            border-top: 150px solid $white;
+            border-left: 150px solid $white;
+            border-bottom: 150px solid transparent;
+            border-right: 150px solid transparent;
+            z-index: 1;
+        }
+        .card-title-text{
+            position: absolute;
+            font-size: $fontsize-l;
+            font-weight: 700;
+            margin-top: 0;
+            margin-left: 4px;
```

```suggestion
            top: 0;
            left: 4px;
```
position: absolute の時は絶対位置指定を使って位置を調整していきましょう！